pkg/gitparse/gitparse.go

```diff
@@ -40,10 +42,86 @@ type Commit struct {
 
 // Diff contains the info about a file diff in a commit.
 type Diff struct {
-	PathB     string
-	LineStart int
-	Content   bytes.Buffer
-	IsBinary  bool
+	PathB             string
+	LineStart         int
+	Content           bytes.Buffer // Keep in-memory buffer for smaller diffs
+	streamDestination *os.File     // File destination for larger diffs
```

Comment:
Should `Content` be unexported?

Reply:
I tried, but apparently it's used elsewhere in the code :(. Tried not to touch too many more things with this change, but I think it's something we can look to change.

Reply:
`Content` doesn't always have the diff contents though, so if it's being used elsewhere that sounds like a bug waiting to happen.

Reply:
My thinking is, if it's currently an issue, this won't make it worse. 😅